### PR TITLE
PSP-1433 Agency error appears when submitting access request

### PR DESCRIPTION
**Pull request by asanchezr**
MOTI has no use for agencies so they have been removed from relevant forms and API endpoints. The database schema has been left as-is (no DB changes)

* Remove all references to "agency" on frontend forms (Access Request page & Edit User page)
* The API no longer checks for non-empty agencies array before creating access requests
* Update unit tests to reflect this change

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -59,7 +59,15 @@ stop: ## Stops the local containers (n=service name)
...
-	@docker-compose build --no-cache $(n)
+	@docker-compose build --force-rm --no-cache $(n)
+
+build-fe: ## Rebuilds the frontend container
+	@echo "$(P) Building frontend..."
+	@make build n=frontend
+
+build-be: ## Rebuilds the backend container
+	@echo "$(P) Building backend..."
+	@make build n=backend
```
Comment (asanchezr) [inline Makefile:70]: convenience shortcuts so I don't have to type long **make** commands

File: backend/api/Areas/Admin/Controllers/AccessRequestController.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ public class AccessRequestController : Controller
         private readonly IUserService _userService;
         #endregion
 
-        #region Construstor
+        #region Constructor
```
Comment (asanchezr) [inline backend/api/Areas/Admin/Controllers/AccessRequestController.cs:32]: typo

File: backend/api/Controllers/UserController.cs [hunk 3]
```diff
@@ -120,14 +120,10 @@ public IActionResult GetAccessRequest(int id)
         [SwaggerOperation(Tags = new[] { "user" })]
         public async Task<IActionResult> AddAccessRequestAsync([FromBody] Model.AccessRequestModel model)
         {
-            if (model == null || model.Agencies == null || model.Roles == null)
```
Comment (asanchezr) [inline backend/api/Controllers/UserController.cs:123]: Removed checks for null and empty agencies on relevant API endpoints

File: backend/api/Controllers/UserController.cs [hunk 4]
```diff
@@ -120,14 +120,10 @@ public IActionResult GetAccessRequest(int id)
...
+            if (model == null || model.Roles == null)
             {
                 throw new BadRequestException("Invalid access request specified");
             }
-            if (model.Agencies.Count() != 1)
-            {
-                throw new BadRequestException("Each access request can only contain one agency.");
-            }
```
Comment (asanchezr) [inline backend/api/Controllers/UserController.cs:130]: We no longer have a need for this check. The frontend will always send null agencies

File: frontend/src/features/admin/access-request/AccessRequestPage.CITZ.test.tsx [hunk 5]
```diff
@@ -1,231 +0,0 @@
-import { useKeycloak } from '@react-keycloak/web';
```
Comment (asanchezr) [inline frontend/src/features/admin/access-request/AccessRequestPage.CITZ.test.tsx:1]: This test file is no longer relevant as we are not doing alignment anymore

File: frontend/src/features/admin/access-request/AccessRequestPage.tsx [hunk 6]
```diff
@@ -63,37 +59,15 @@ const AccessRequestPage = () => {
...
     rowVersion: accessRequest?.rowVersion,
   };
 
-  // PSP-722 Disable agency dropdown from Access Request Form (only for MOTI)
-  const tenant = useTenant();
-  if (tenant?.id === 'MOTI') {
-    initialValues.showAgency = false;
-  }
-
-  const selectAgencies = agencies.map(c => mapLookupCode(c, initialValues.agency));
```
Comment (asanchezr) [inline frontend/src/features/admin/access-request/AccessRequestPage.tsx:82]: Deleting extra logic that is no longer needed

File: frontend/src/features/admin/edit-user/EditUserPage.test.tsx [hunk 7]
```diff
@@ -114,20 +114,9 @@ describe('Edit user page', () => {
...
-  it('displays enabled agencies', () => {
-    const { queryByText } = testRender();
-    expect(queryByText('agencyVal')).toBeVisible();
-  });
-
-  it('Does not display disabled agencies', () => {
-    const { queryByText } = testRender();
-    expect(queryByText('disabledAgency')).toBeNull();
-  });
-
```
Comment (asanchezr) [inline frontend/src/features/admin/edit-user/EditUserPage.test.tsx:130]: Remove unnecessary tests

File: frontend/src/utils/YupSchema.ts [hunk 8]
```diff
@@ -14,13 +14,6 @@ Yup.addMethod(Yup.string, 'optional', function optional() {
 });
 
 export const AccessRequestSchema = Yup.object().shape({
-  showAgency: Yup.boolean(),
-  agency: Yup.number().when('showAgency', {
-    is: true,
-    then: Yup.number()
-      .min(1, 'Invalid Agency')
-      .required('Required'),
-  }),
```
Comment (asanchezr) [inline frontend/src/utils/YupSchema.ts:23]: Remove validation for agencies field in Access Request page

File: frontend/src/features/admin/access/__snapshots__/ManageAccessRequests.test.tsx.snap [hunk 9]
```diff
@@ -275,7 +275,7 @@ exports[`Manage access requests Snapshot matches 1`] = `
             Object {
               "display": "flex",
               "flex": "1 0 auto",
-              "minWidth": "780px",
+              "minWidth": "580px",
```
Comment (devinleighsmith) [inline frontend/src/features/admin/access/__snapshots__/ManageAccessRequests.test.tsx.snap:278]: Where did this change come from?

**Review by devinleighsmith — APPROVED** [review]

**Review by Fosol — COMMENTED** [review]
While we will be losing `Agency` we will be replacing it with `Organization`.  Which means much of this logic will most likely need to be re-added.

File: frontend/src/features/admin/access/__snapshots__/ManageAccessRequests.test.tsx.snap (hunk 9 above)
Comment (asanchezr) [reply frontend/src/features/admin/access/__snapshots__/ManageAccessRequests.test.tsx.snap:278]: From removing the **Agency** column from the list view in Manage Access Requests

**Comment by asanchezr** [comment]
@Fosol I think I rather re-add the new functionality when needed than keeping error prone code in place just waiting to be refactored

**Review by Fosol — APPROVED** [review]

**Pull request merged**